src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/DiagnosticScopeCodeRefactoringProviderTest.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Xunit;
+using Verifier = Azure.ClientSdk.Analyzers.Tests.AzureRefactoringVerifier<Azure.ClientSdk.Analyzers.DiagnosticScopeCodeRefactoringProvider>;
+
+namespace Azure.ClientSdk.Analyzers.Tests
+{
+    public class DiagnosticScopeCodeRefactoringProviderTest
```

Comment:
Would be good to have some negative tests for the cases when refactoring shouldn't be suggested.